Akkadu/rsi-widgets
# subtitles-feature

Please describe the purpose of the pull request.

* The purpose was to implement our subtitles feature on the SDK.
* I made a new package called: `rsi-subtitles` that only handle the UI logic.
* On the interpretation-player I listen to the RTM-messages (the subtitles). When I switch languages on the `interpretation-player`, I push to the dom ID `akkadu-rsi-subtitles`.

## Motivation and Context



## How To Test



* On akkadu.com
* Event name : MULTI-LANGUAGE-2
* Interpreter accounts:
1- French : romain@akkadu-team.com
`akkadu_team_1234`



* On the Vue or React repo you should be able to 
  * 1-Switch to the language you are currently interpreting and speak. you should see the subtitle being generated.




## Screenshots (if appropriate)

## Checklist



加油！
